src/core/prob_macro.jl

```diff
@@ -0,0 +1,186 @@
+using .RandomVariables: VarInfo, TypedVarInfo, Metadata, setval!, settrans!
+using MCMCChains: Chains
+using Turing: PriorContext, LikelihoodContext, SampleFromPrior
+
+macro logprob_str(str)
+	expr1, expr2 = get_exprs(str)
+    return :(Turing.Core.logprob($expr1, $expr2)) |> esc
+end
+macro prob_str(str)
+	expr1, expr2 = get_exprs(str)
+    return :(exp.(Turing.Core.logprob($expr1, $expr2))) |> esc
+end
+
+function get_exprs(str::String)
```

Comment:
It feels like it might be necessary to add some error handling here.

Reply:
Especially if `|` is not existing.

Reply:
I agree.